athenet/data_loader/data_loader.py

```diff
@@ -28,75 +29,77 @@ def batch_size(self, value):
         self.n_val_batches = self.val_set_size / self.batch_size
         self.n_test_batches = self.test_set_size / self.batch_size
 
+    def _set_subset(self, data, value, index, size=1):
+        data[index*self.batch_size:(index+size)*self.batch_size] = value
+
     def _get_subset(self, data, index, size=1):
-        return data[index*self.batch_size:
-                    (index+size)*self.batch_size]
+        return data[index*self.batch_size:(index+size)*self.batch_size]
 
     def train_input(self, batch_index):
```

Comment:
Is NotImplementedError a target behaviour in methods below, or they are just TODO? Put a comment on that.